app/assets/stylesheets/elements/_table.scss

```diff
@@ -74,8 +74,6 @@ table tbody {
 
   td h3 {
     border-bottom: 0.125rem solid $color-gray-light;
-    color: $color-green-light;
```

Comment:
Does this change affect any other pages?

Reply:
No cause it's in the .special-items-table {} block